webpack.config.js

```diff
@@ -44,7 +44,9 @@ var settings = {
       }),
       new webpack.NamedModulesPlugin(),
       new CopyWebpackPlugin([
-         { from: 'src/js/dependencies/gzip.js', to: 'js/' }
+         { from: 'src/js/dependencies/gzip.js', to: 'js/' },
+         { from: 'example-experiments/lexical-decision.osexp', to: 'osexp/' },
+         { from: 'example-experiments/gaze-cuing.osexp', to: 'osexp/' }
```

Comment:
You should be able to use wildcards here ([docs](https://github.com/kevlened/copy-webpack-plugin)), so one line with
~~~
{ from: 'example-experiments/*.osexp', to: 'osexp/' },
~~~
should suffice.